src/sass/icons/_icons-sizes.scss

```diff
@@ -7,37 +7,37 @@ Markup:
 </svg>
 
 icon-xs      - Xs
-icon-small   - Small
-icon-favicon - Favicon
-icon-normal  - Normal
-icon-medium  - Medium
 icon-sm      - Sm
 icon-md      - Md
+icon-normal  - Normal
 icon-lg      - Lg
-icon-xxl     - Xxl
+icon-2x      - 2x
+icon-3x      - 3x
+icon-4x      - 4x
 
-Weight: 6
+Weight: 13
 
 Styleguide Icons.Icons-Sizes
 */
 
 $iconsSizes:
-  ('xs' , 10),
-  ('small', 13),
-  ('favicon', 16),
+  ('xs', 13),
+  ('sm', 16),
+  ('md', 20),
   ('normal', 24),
-  ('medium', 25),
-  ('sm', 30),
-  ('md', 60),
-  ('lg', 90),
-  ('xxl', 120);
+  ('lg', 32),
+  ('2x', 48),
+  ('3x', 64),
+  ('4x', 128);
 
 .icon {
   background-image: none;
   display: inline-block;
-  height: 20px;
+  height: 24px;
```

Comment:
This may have a lot of implications on the icons with no size explicitly set, not sure if it's a good idea.